keyboards/cipulot/kawayo/readme.md

```diff
@@ -0,0 +1,29 @@
+# Kawayo
+
+![Kawayo Board](https://cdn.shopify.com/s/files/1/0281/3122/9834/products/Kawayoo_1024x1024@2x.jpg?v=1650025682)
```

Comment:
Please rehost this on Imgur or a similar image hosting service. 

Shopify URLs are notoriously fickle... and disappear all the time.  So we really want/need a long term solution.

Reply:
@drashna changes made. Switched to Imgur hosting.